### PR TITLE
Add logger config

### PR DESCRIPTION
**Pull request by myslak71**

### CODE REVIEW

File: src/colejorz/__init__.py [hunk 1]
```diff
@@ -13,25 +15,37 @@ def get_stationmaster(request):
     return request.registry.stationmaster
 
 
-def serve(**settings):
+def app_factory(global_config, **settings):  # pylint:disable=unused-argument
```
Comment (myslak71) [inline src/colejorz/__init__.py]: `global_config` is required by PasteDeploy. Same in line 34.

File: src/colejorz/__init__.py [hunk 2]
```diff
@@ -13,25 +15,40 @@ def get_stationmaster(request):
     return request.registry.stationmaster
 
 
-def serve(**settings):
+def app_factory(
+        global_config: OrderedDict,  # pylint:disable=unused-argument
```
Comment (myslak71) [inline src/colejorz/__init__.py:20]: `global_config` is required by PasteDeploy. Same in line 37.

**Comment by fizyk** [comment]
Also update bin/ccolejorz.service

**Comment by myslak71** [comment]
> Also update bin/ccolejorz.service

**Pull request merged**